### ISSUE TITLE
scrape produces error

### ISSUE THREAD

**Issue by swiftaw** [cpsievert/pitchRx#73]
Every scrape call results in the following error message: `Error in function (type, msg, asError = TRUE)  : `

**Comment by cpsievert** [comment]
Does `remotes::install_github("cpsievert/pitchRx")`  help at all?

**Comment by Mpellet771** [comment]
It doesn't help. There's some request to update the gids. How do we do that?

**Comment by Katiedaisey** [comment]
I'm working on fixing this for the new MLB api. Does anyone have some example data they pulled previously so I can keep the format consistent?

**Comment by Mpellet771** [comment]
I have an SQLlite file that may help. Don't have any raw downloads.

**Comment by Katiedaisey** [comment]
@Mpellet771 good place to start I guess. If you could send it to me or put it up on your GitHub, I’ll fork it. Thanks

**Comment by Mpellet771** [comment]
Hey, actually, the MLB shut down the gd2 (gameday) server. We now have to go to mlbstats. See this article: https://www.npmjs.com/package/mlb-stats-api

**Comment by Katiedaisey** [comment]
Yup, I’ve already pulled data from the new site and have their framework mapped. I just need to know what to map it to.

**Comment by Mpellet771** [comment]
Great! I've added the create tables set of commands to my repository (the actual database is too big). See here: [Mpellet771](https://github.com/Mpellet771)
/
[BaseballSQL](https://github.com/Mpellet771/BaseballSQL)
